CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ find_package(podio 0.15 REQUIRED)
 include_directories(${podio_INCLUDE_DIR})
 
 find_package(EDM4HEP 0.4.1 REQUIRED)
+include_directories(${EDM4HEP_INCLUDE_DIR})
```

Comment:
[`target_include_directories`](https://cmake.org/cmake/help/latest/command/target_include_directories.html), please!

Reply:
I would indeed prefer implementing it this way, but it adds fragility. The name of the target is specified by the macros provided by the upstream podio project, and is subject to changes there (i.e. this is an issue that should be resolved upstream and a bug report is in progress). In any case, we can use `target_include_directories` and deal with future failures due to upstream changes if and when they happen.

Reply:
At least the build failures that would be introduced by upstream changes should show up right when someone is making those version changes, so the correlation will be clear.

Reply:
Right. If we don't control the name of the cmake target, we are in much bigger trouble downstream.